agent-scripts/readme.md

```diff
@@ -0,0 +1,23 @@
+#enabling nginx stats to be read by zabbix_check_nginx-status:
+put following configuration on your nginx.conf file:
+
+```
+location /basic_status {
+stub_status on;
+access_log off;
+allow 127.0.0.1;
+deny all;}
+```
+and reload the nginx
+
+
+#Copy the files 
+```
+https://github.com/hamidsfandiari/zabbix-agent-extensions/blob/patch-2/agent-scripts/tools/zabbix_check_nginx-status
```

Comment:
Why do you need that? The copy procedure should not be neccessary if you use rpm or deb packages.